pkg/scalers/azure/azure_eventhub_checkpoint.go

```diff
@@ -35,17 +35,15 @@ import (
 // goCheckpoint struct to adapt goSdk Checkpoint
 type goCheckpoint struct {
 	Checkpoint struct {
-		SequenceNumber int64  `json:"sequenceNumber"`
-		Offset         string `json:"offset"`
+		SequenceNumber int64 `json:"sequenceNumber"`
 	} `json:"checkpoint"`
 	PartitionID string `json:"partitionId"`
 }
 
 type baseCheckpoint struct {
-	Epoch  int64  `json:"Epoch"`
-	Offset string `json:"Offset"`
-	Owner  string `json:"Owner"`
-	Token  string `json:"Token"`
+	Epoch int64  `json:"Epoch"`
+	Owner string `json:"Owner"`
+	Token string `json:"Token"`
 }
```

Comment:
I'm not sure if we still need this struct for something. I mean, its fields aren't used either

Reply:
Removed